src/ngraph/op/experimental/layers/psroi_pooling.hpp

```diff
@@ -47,6 +47,11 @@ namespace ngraph
             virtual std::shared_ptr<Node>
                 copy_with_new_args(const NodeVector& new_args) const override;
 
+            size_t get_output_dim() const { return m_output_dim; }
```

Comment:
Any deep reason for not using a `struct` on this one?

Reply:
Not really. In general, if the number of attributes crosses a certain pain threshold, I packed them in a struct.